tests/lib/managers/terms-of-service-test.js

```diff
@@ -0,0 +1,571 @@
+		TermsOfService = require(MODULE_FILE_PATH);
+		termsOfService = new TermsOfService(boxClientFake);
+	});
+
+	afterEach(function() {
+		sandbox.verifyAndRestore();
+		mockery.deregisterAll();
+		mockery.disable();
+	});
+
+	describe('get()', function() {
+		it('should make GET request to get terms of service info when called with ID', function() {
+			var response = {
+				type: 'terms_of_service',
+				id: '1234',
+				status: 'enabled',
+				enterprise: {},
+				tos_type: 'external',
+				text: 'TEST',
+				created_at: '2017-10-21T20:09:59-07:00',
+				modified_at: '2017-10-21T20:09:59-07:00'
+			};
+			sandbox.stub(boxClientFake, 'wrapWithDefaultHandler').returnsArg(0);
+			sandbox.mock(boxClientFake).expects('get').withArgs('/terms_of_services/1234')
+				.returns(Promise.resolve(response));
+			termsOfService.get(TERMS_OF_SERVICE_ID);
+		});
+
+		it('should call callback with the terms of service info response is returned', function(done) {
+			var response = {
+				type: 'terms_of_service',
+				id: '1234',
+				status: 'enabled',
+				enterprise: {},
+				tos_type: 'external',
+				text: 'TEST',
+				created_at: '2017-10-21T20:09:59-07:00',
+				modified_at: '2017-10-21T20:09:59-07:00'
+			};
+			sandbox.stub(boxClientFake, 'wrapWithDefaultHandler').returnsArg(0);
+			sandbox.stub(boxClientFake, 'get').yieldsAsync(null, response);
+			termsOfService.get(TERMS_OF_SERVICE_ID, null, function(err, termsOfServiceObject) {
+				assert.ifError(err);
+				assert.strictEqual(termsOfServiceObject, response, 'terms of service object is returned');
+				done();
+			});
+		});
+
+		it('should return a promise resolving to a terms of service object when a 200 response is returned', function() {
+			var response = {
+				type: 'terms_of_service',
+				id: '1234',
+				status: 'enabled',
+				enterprise: {},
+				tos_type: 'external',
+				text: 'TEST',
+				created_at: '2017-10-21T20:09:59-07:00',
+				modified_at: '2017-10-21T20:09:59-07:00'
+			};
+			sandbox.stub(boxClientFake, 'wrapWithDefaultHandler').returnsArg(0);
+			sandbox.stub(boxClientFake, 'get').returns(Promise.resolve(response));
+			return termsOfService.get(TERMS_OF_SERVICE_ID)
+				.then(termsOfServiceObject => {
+					assert.strictEqual(termsOfServiceObject, response, 'terms of service object is returned');
+				});
+		});
+	});
+
+
+	describe('getAll()', function() {
+		it('should make GET request to get terms of service info when called', function() {
+			var response = {
+				total_count: 2,
+				entries: [{
+					type: 'terms_of_service',
+					id: '1234',
+					status: 'enabled',
+					enterprise: {},
+					tos_type: 'external',
+					text: 'TEST',
+					created_at: '2017-10-21T20:09:59-07:00',
+					modified_at: '2017-10-21T20:09:59-07:00'
+				}]
+			};
+			sandbox.stub(boxClientFake, 'wrapWithDefaultHandler').returnsArg(0);
+			sandbox.mock(boxClientFake).expects('get').withArgs('/terms_of_services')
+				.returns(Promise.resolve(response));
+			termsOfService.getAll();
```

Comment:
For completeness, we should pass options here and make sure they get passed through to `get()`